tesseract_kinematics/kdl/src/kdl_inv_kin_chain_lma.cpp

```diff
@@ -69,7 +72,7 @@ KDLInvKinChainLMA::KDLInvKinChainLMA(const KDLInvKinChainLMA& other) { *this = o
 KDLInvKinChainLMA& KDLInvKinChainLMA::operator=(const KDLInvKinChainLMA& other)
 {
   kdl_data_ = other.kdl_data_;
-  ik_solver_ = std::make_unique<KDL::ChainIkSolverPos_LMA>(kdl_data_.robot_chain);
+  ik_solver_ = other.ik_solver_;
```

Comment:
This need to be reverted because it causes issues with mult threading. They need to be unique per object.